app/src/main/java/Script/IScriptProvider.java

```diff
@@ -1,6 +1,9 @@
 package Script;
 
-public interface IScriptProvider {
+import org.json.JSONObject;
+
+public interface IScriptProvider
+{
```

Comment:
I won't keep commenting on these, but please remove all similar changes. (I'm open to debating this, by the way, but we need to settle it before I merge this change set.)